tests/test_release_transpiler_argument_types.py

```diff
@@ -0,0 +1,330 @@
+import os
+from os.path import abspath, dirname
+
+import pytest
+
+RELEASE_TRANSPILER_FILE_PATH = abspath(
+    dirname(dirname(abspath(__file__))) + "/komodo/release_transpiler.py"
+)
+VALID_RELEASE_FOLDER = abspath(dirname(__file__) + "/data/test_releases")
+VALID_RELEASE_BASE = "2020.01.a1"
+VALID_OVERRIDE_MAPPING_FILE = abspath(
+    dirname(dirname(__file__)) + "/examples/stable.yml"
+)  # "/Users/JONAK/Documents/FMU/SCOUT/komodo/examples/stable.yml"
+VALID_PYTHON_COORDS = "3.6,3.8"
+
+
+# TESTING COMBINER
+@pytest.mark.parametrize(
+    "args, expectedExitCode",
+    [
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords 3.8,3.6"
+            ),
+            0,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE}"
+            ),
+            0,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords py3.8"
+            ),
+            512,
```

Comment:
This magic number should probably be defined in a constant with a descriptive name.